cannythreshold1: 400
cannythreshold2: 1000
houghRho: 1
houghTheta: rad(pi/180)
houghThreshold: 50
houghMinLineLength: 50
houghMaxLineGap: 50